Marlin/ultralcd_impl_DOGM.h

```diff
@@ -591,6 +591,29 @@ static void lcd_implementation_status_screen() {
     u8g.setPrintPos(12, 50);
     lcd_print(itostr3(feedrate_percentage));
     u8g.print('%');
+
+    //
+    // Filament sensor if SD is disabled
+    //
+
+    #if DISABLED(SDSUPPORT) && ENABLED(FILAMENT_LCD_DISPLAY)
+      // Show measured value
+      lcd_setFont(FONT_MENU);
+      u8g.setPrintPos(47, 50);
+      lcd_print('\xf8'); // TODO: LCD_STR_FILAMENTSIZE
+      lcd_setFont(FONT_STATUSMENU);
+      u8g.setPrintPos(56, 50);
+      lcd_print(ftostr12ns(filament_width_meas));
+
+      // Show calculated multiplier
+      lcd_setFont(FONT_MENU);
+      u8g.setPrintPos(93, 50);
+      lcd_print("\xa4"); // TODO: LCD_STR_FILAMENTMULTIPLIER
+      lcd_setFont(FONT_STATUSMENU);
+      u8g.setPrintPos(102, 50);
+      lcd_print(itostr3(100.0 * volumetric_multiplier[FILAMENT_SENSOR_EXTRUDER_NUM]));
+      u8g.print('%');
+    #endif
```

Comment:
For efficiency this should also follow the example of the XYZ coordinates, whose strings are guaranteed to be calculated only once, when `page.page == 0`.
```cpp
static char xstring[5], ystring[5], zstring[7];
#if ENABLED(FILAMENT_LCD_DISPLAY) && DISABLED(SDSUPPORT)
  static char wstring[5], mstring[4];
#endif

// At the first page, regenerate the XYZ strings
if (page.page == 0) {
  strcpy(xstring, ftostr4sign(current_position[X_AXIS]));
  strcpy(ystring, ftostr4sign(current_position[Y_AXIS]));
  strcpy(zstring, ftostr52sp(current_position[Z_AXIS] + 0.00001));
  #if ENABLED(FILAMENT_LCD_DISPLAY) && DISABLED(SDSUPPORT)
    strcpy(wstring, ftostr12ns(filament_width_meas));
    strcpy(mstring, itostr3(100.0 * volumetric_multiplier[FILAMENT_SENSOR_EXTRUDER_NUM]));
  #endif
}
```
…which leads to…
```cpp
#if DISABLED(SDSUPPORT) && ENABLED(FILAMENT_LCD_DISPLAY)
  // Show measured value
  u8g.setPrintPos(56, 50);
  lcd_print(wstring);
  // Show calculated multiplier
  u8g.setPrintPos(102, 50);
  lcd_print(mstring);
  u8g.print('%');
  lcd_setFont(FONT_MENU);
  u8g.setPrintPos(47, 50);
  lcd_print('\xf8'); // TODO: LCD_STR_FILAMENTSIZE
  u8g.setPrintPos(93, 50);
  lcd_print("\xa4"); // TODO: LCD_STR_FILAMENTMULTIPLIER
#endif
```